core/blockchain_repair_test.go

```diff
@@ -132,252 +126,10 @@ func TestShortFastSyncingRepair(t *testing.T) {
 	// Expected head block     : G
 	testRepair(t, &rewindTest{
 		canonicalBlocks:    8,
-		sidechainBlocks:    0,
 		freezeThreshold:    16,
 		commitBlock:        0,
 		pivotBlock:         uint64ptr(4),
 		expCanonicalBlocks: 8,
-		expSidechainBlocks: 0,
```

Comment:
Why do these values need to be deleted?

Reply:
We are explicitly failing if something is inserted as a sidechain, and these checks, basically ask for the amount of blocks as a sidechain.
I thought of having all of those with 0, but, as I'm trying to remove the concept of sidechain, if I let those, in the future I think that will only generate some noice. This way it's a clearer test